### DISCUSSION TITLE
How to list available versions of any package?

### DISCUSSION THREAD

**Discussion by dai1975** [aquaproj/aqua#2182] [Deprecated (Question and Answer)] [answered]
### Question

"aqua list" lists available packages but I cannot get version list of any package. How to do it?

### Background of the question

When I manually edit aqua.yaml to install software, I wonder which version is supported by aqua.

By googling, I can do it by "aqua g -s" but in my env fuzzy finder failed because of "character set not supported".
It not a problem, I prefer to use UNIX shell stdout way.

### Example Code

```console
$ aqua list -h
OPTIONS:
   --help, -h  show help
```

### Reference

for information, the asdf command can do it:

```
$ asdf list all golang | head
1.2.2
1.3
1.3beta1
1.3beta2
1.3rc1
1.3rc2
1.3.1
1.3.2
1.3.3
1.4
```

**Comment by suzuki-shunsuke** [comment, marked as answer]
Thank you for your question.

aqua doesn't support outputting available versions to stdout for now.

> By googling, I can do it by "aqua g -s" but in my env fuzzy finder failed because of "character set not supported".

I have never seen the same error. It's interesting.
Could you tell me the result of `aqua info` and how to reproduce the error? Does this error happen with only specific packages?

```console
$ aqua info
```

```console
$ aqua g -s
```

**Comment by dai1975** [comment]
```
$ aqua info
{
  "version": "2.10.1",
  "commit_hash": "fbf3b157b21897fa2f315fbc30474acafded108d",
  "os": "linux",
  "arch": "arm64",
  "pwd": "/home/(USER)",
  "root_dir": "/home/(USER)/.local/share/aquaproj-aqua",
  "env": {
    "AQUA_GLOBAL_CONFIG": "/home/(USER)/.config/aquaproj-aqua/aqua.yaml"
  },
  "config_files": []
}

$ aqua g -s
FATA[0000] aqua failed                                   aqua_version=2.10.1 env=linux/arm64 error="find the package: failed to initialize the fuzzy finder: failed to initialize screen: character set not supported" program=aqua
```

I think this error is caused because of my environment.
Zsh on tmux on ssh login to Ubuntu on QEMU(without amd64 emuration) on M1  Mac.
I doubt TERM environment is not set well, but I want not take time to solve it.
